static/xref/script.js

```diff
@@ -180,10 +180,11 @@ function howToCiteMarkup(term, entry) {
 
 function howToCiteTerm(term, entry) {
   const { type, for: forList, shortname } = entry;
+  term = term.replace('/', '\\/');
   if (forList) {
     return forList.map(f => `[=${f}/${term}=]`).join('<br>');
   }
-  return `&lt;a data-cite="${shortname}">${term}&lt;/a>`;
+  return `[=${term}=]`;
```

Comment:
Question... remember can't match the "sRGB" because case?... in Shepherd, are terms stored as they are exported? If so, when  recommending the term, maybe this term can be whatever has been exported... so, if I query "srgb", how to cite should be  `[=sRGB=]`.

Reply:
Bikeshed-data has terms as lowercase. But I'll look into something to get original terms